dashboard/app/models/rubric_ai_evaluation.rb

```diff
@@ -28,4 +28,22 @@ class RubricAiEvaluation < ApplicationRecord
   has_many :learning_goal_ai_evaluations, inverse_of: :rubric_ai_evaluation
 
   validates :status, inclusion: {in: SharedConstants::RUBRIC_AI_EVALUATION_STATUS.values}
+
+  def summarize_debug
```

Comment:
what's this used for?

Reply:
This is just to help with debugging, I found it useful for learning goal ai evaluation so I left it in for future use